c++/Search in a row-column sorted Matrix.cpp

```diff
@@ -0,0 +1,21 @@
+int search( int n,int m, int x, int mat[SIZE][SIZE])
```

Comment:
Hi, please have a look at the [contribution tips](https://github.com/omonimus1/geeks-for-geeks-solutions#contribution-essentials-note).

Reply:
Okay, and sorry for not looking for that before :)